packages/ember-runtime/lib/mixins/action_handler.js

```diff
@@ -209,6 +209,20 @@ var ActionHandler = Mixin.create({
                    ') does not have a `send` method', typeof target.send === 'function');
       target.send(...arguments);
     }
+  },
+
+  /**
+   Check if controller has any subscriptions for named event.
```

Comment:
"named event" or "specific action" or "an action, by name" - seems like the doc could use more description.